src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10Driver.cs

```diff
@@ -67,10 +67,13 @@ protected internal override void AddCallbackForPinValueChangedEvent(int pinNumbe
         /// <param name="pinNumber">The pin number in the driver's logical numbering scheme.</param>
         protected internal override void ClosePin(int pinNumber)
         {
-            if (_openPins.TryGetValue(pinNumber, out Windows10DriverPin pin))
+            if (_openPins.TryGetValue(pinNumber, out Windows10DriverPin? pin))
             {
-                pin.ClosePin();
-                _openPins.Remove(pinNumber);
+                if (pin is object)
```

Comment:
I don't think you need this check, as we can guarantee that if TryGetVaue is true then pin won't be null. Where you getting a warning because of this?

Reply:
If you were, I would probably just use exclamation point rather than make that extra check since we know it will be defined